app/src/main/java/com/github/se/eventradar/ui/login/LoginScreen.kt

```diff
@@ -55,57 +86,94 @@ fun LoginScreen(navigationActions: NavigationActions) {
           .setIsSmartLockEnabled(false)
           .setAvailableProviders(providers)
           .build()
-
   Column(
       modifier = Modifier.fillMaxSize().testTag("loginScreen"),
       verticalArrangement = Arrangement.Center,
       horizontalAlignment = Alignment.CenterHorizontally,
   ) {
-    Image(
-        painter = painterResource(R.drawable.image1),
-        contentDescription = "Logo",
-        modifier = Modifier.width(132.dp).height(132.dp).testTag("logo"),
-    )
-    Text(
-        text = "Welcome",
-        style =
-            TextStyle(
-                fontSize = 57.sp,
-                fontFamily = FontFamily.SansSerif,
-                fontWeight = FontWeight(400),
-                textAlign = TextAlign.Center,
-            ),
-        modifier = Modifier.testTag("loginTitle"))
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
```

Comment:
You should modify the color to work with both light and dark mode (using the material design colors).

Reply:
👍